toolkits/global/packages/global-forms/scss/50-components/_forms.scss

```diff
@@ -43,6 +43,19 @@
 	max-width: $global-forms-input-width-maximum;
 }
 
+.c-forms__fieldset--pictographic-radios {
+	display: flex;
```

Comment:
Is there anything that motivates to not go mobile first?

Reply:
I must admit I have been a bit bad on this and started with desktop. However, saying that, I think these styles are written in a mobile first way, no? It's not clear but `@include media-query('xs') { }` actually means it applies to anything xs size and above.

Reply:
Actually what I usually do is to write the props that needs to be for served to the mobile viewport without surrounding media queries, and then override the relevant ones by surrounding them with appropriate media-query to cater for larger viewports

Reply:
Here a little snippet stolen for the World Wide Web:

```css
// This applies from 0px to 600px
body {
  background: red;
}

// This applies from 600px onwards
@media (min-width: 600px) {
  body {
    background: green;
  }
}
```

Reply:
Oh sorry, it may have been called `from-xs` in past versions. Apologies, then all 👌